ITE/4/README.adoc

```diff
@@ -233,22 +232,26 @@ properties. For example, an implementation that is capable of hashing GitHub
 entities may choose to use a subsection of the contents obtained using the
 official API, which returns JSON representations containing different properties
 of the entities. It is important for these representations to be consistent for
-a particular entity and they must be generated keeping in mind the different
-sources of non-determinism as highlighted in the Appendix. It's also necessary
-for this method to handle missing resources safely.
-
-Being able to get some hashable representation of abstract entities is necessary
-for in-toto inspections to verify contents as the context demands it. Therefore,
-it should be possible for implementations to use the method in scenarios other
-than just the recording of hashes of an artifact.
-
-It is quite likely that a generic URI specified may point to a collection of
+a particular entity, and to acknowledge the different
+sources of non-determinism highlighted in the Appendix. It is also necessary
+for this method to handle any missing resources safely. That is, if a resource does not exist
+at the location pointed to by a URI, this method must fail in a secure manner.
+
+Being able to get some hashable representation of abstract entities is also needed
+for in-toto inspections to verify contents. There is no specification mandating
+how parts of the supply chain are to be verified. Yet, these abstract resources still
+need to be resolvable so that they can be inspected if the supply chain owner so decides to.
+Therefore,
+it should be possible for implementations to use get_hashable_representation in scenarios other
```

Comment:
get_hashable_respresentation should enclosed in backticks - \`get_hashable_representation\`

Reply:
Please make the suggestion and I will accept it.

Reply:
Resolved. (I'm unable to mark it for some reason)